### PR TITLE
Add date in message icon

### PR DESCRIPTION
**Pull request by charlietango**
Fix for monitorizare-vot-votanti-client#45

screenshot 2019-01-26 at 14 19 04

### CODE REVIEW

File: app/components/Icons/icons/icon-it_election_day.js [hunk 1]
```diff
@@ -1,17 +1,25 @@
 import React from 'react';
 
-function IconItElectionDay() {
+function IconItElectionDay(props) {
+  const createdAtDate = props.createdAt ? props.createdAt.split(' ')[0] : '';
+  const month = createdAtDate && createdAtDate.split('-')[1];
+  const day = createdAtDate && createdAtDate.split('-')[2];
```
Comment (danichim) [inline app/components/Icons/icons/icon-it_election_day.js]: daca se schimba format-ul va fi problema

Comment (charlietango) [reply app/components/Icons/icons/icon-it_election_day.js]: Good point

**Comment by danichim** [comment]
Aici confuzezi data cu luna nu stii exact, nu e prea intuitiv.

**Comment by aniri** [comment]
check changes required in https://github.com/code4romania/monitorizare-vot-votanti-client/issues/45

**Comment by RaduCStefanescu** [comment]
Merged this change with changes from another pull request, addressing the same issue and we merged the other one, please check https://github.com/code4romania/monitorizare-vot-votanti-client/pull/48
Will close this pull request, thank you very much for the work, we really appreciate it!

**Pull request closed without merging**